js/renovation/widget.tsx

```diff
@@ -16,6 +16,7 @@ import { each } from '../core/utils/iterator';
 import { extend } from '../core/utils/extend';
 import { focusable } from '../ui/widget/selectors';
 import { isFakeClickEvent } from '../events/utils';
+import { useLayoutEffect } from 'preact/hooks';
```

Comment:
Should we use something like `@layoutEfffect`?

Reply:
I don't think so, there should be strong reason to use LayoutEffect instead of just Effect

Reply:
Okay